google/cloud/vision_helpers/__init__.py

```diff
@@ -26,7 +27,7 @@ class VisionHelpers(object):
     See the :class:`~google.cloud.vision_v1.ImageAnnotatorClient`.
     """
 
-    def annotate_image(self, request, retry=None, timeout=None):
+    def annotate_image(self, request, retry=None, timeout=None, metadata=()):
```

Comment:
I think that retry, timeout, and metadata should be keyword only to maintain consistency with the generated surface. Other people may disagree, and their opinions are probably more relevant.

Reply:
+1 to this. @telpirion or @sirtorry could you also weigh in? New generated clients look like `def some_method(self, request, *, retry=None, timeout=None, metadata=())`. (request positional, everything else keyword only)

Reply:
Line 104, `request = dict(image=image, features=[copied_features], **kwargs)`. All these keywords will be part of the request obejct, so `retry`, `timeout` etc are not in `kwargs`.

Reply:
changed it to `def some_method(self, request, *, retry=None, timeout=None, metadata=())`